styles.css

```diff
@@ -95,6 +111,18 @@ h4 {
   animation: fadeIn 7.8s;
 }
 
+.scroll-down {
+  max-height: 10%;
+  flex-basis: 5%;
+  flex-shrink: 0.5;
+  animation: fadeIn 9.5s ease-out;
+}
+@media (min-width: 1330px) {
+  .scroll-down {
+    display: none;
```

Comment:
Since it doesn't look good on them. Also, there's the scrollbar to indicate scrolling. I changed it so on medium screens you can see the rest of the content.